content/collections/guides_and_surveys/en/custom-css.md

```diff
@@ -0,0 +1,179 @@
+Guides and Surveys should match your brand and feel like part of your product. While [Themes](/docs/guides-and-surveys/themes) provide broad control over appearance, Custom CSS gives you fine-grained control over specific elements using CSS class selectors.
```

Comment:
I think we should be more clear about this being a backup. Something like: Themes provide sophisticated means to change the appearance of your guides and surveys. In sporadic cases, Custom CSS can be a way to exert even greater control. However, try to stick to theme editing wherever possible for greater flexibility and backwards compatibility.

Reply:
That's a good idea. I've updated the intro to frame this as more of a backup.